Project.toml

```diff
@@ -53,6 +54,7 @@ KrylovKit = "0.4.2, 0.5, 0.6, 0.7"
 LinearAlgebra = "1.6"
 LinearMaps = "3"
 NDTensors = "0.2.30"
+Observers = "0.2"
```

Comment:
Perhaps you could try removing this `compat` entry to start out with, maybe that is confusing Julia into thinking it is a dependency instead of a weak dependency.

Reply:
Good idea, I'll try

Reply:
Unfortunately that didn't do it. Maybe I should remove it from extras also just to see.

Reply:
Right, I remember that the only reason I added `VectorInterface` to `[extras]` was so that I could add it to `[compat]`, so that corroborates that might be the primary culprit.

Reply:
Guess not.